charts/argocd-image-updater/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   # -- Default image repository
-  repository: argoprojlabs/argocd-image-updater
+  repository: quay.io/argoprojlabs/argocd-image-updater
```

Comment:
We may want to update the argo-cd values file as well https://github.com/akuity/helm-charts/blob/18fab9ab3426f281f9b3e8a2de601eb1936e3cff/charts/argo-cd/values.yaml#L355

Reply:
Sure, updating. I thought I checked it, good catch!